scripts/cluster/create_one_node_cluster.go

```diff
@@ -0,0 +1,95 @@
+package cluster
```

Comment:
let's first have it separate, make it work. then we can re-factor it in a single file for single and multi-node setup